tests/rptest/services/redpanda.py

```diff
@@ -281,6 +286,17 @@ def clean_node(self, node):
         node.account.remove(f"{RedpandaService.PERSISTENT_ROOT}/*")
         node.account.remove(f"{RedpandaService.CONFIG_FILE}")
 
+    def redpanda_pid(self, node):
+        try:
+            cmd = "ps ax | grep -i 'redpanda' | grep -v grep | awk '{print $1}'"
+            for p in node.account.ssh_capture(cmd,
+                                              allow_fail=True,
+                                              callback=int):
+                return p
+
+        except (RemoteCommandError, ValueError):
+            return None
```

Comment:
is this equivalent to the following?

```python
pids = self.pids(node)
return pids[0] if pids else None
```

Reply:
not necessary since `pids` are not sorted in any way.

Reply:
I'm not sure I understand; it seems to me that neither is sorted. redpanda_pid seems to return the first `p` which is the same as getting all the pids and return `pids[0]`.

The only difference I see is `grep -i 'redpanda\|node'` vs `grep -i 'redpanda'`. How would I know which one of these to use if I were working on the redpanda service?

Reply:
I think this PR is ready to go @mmaslankaprv but can you leave a comment here I think whoever hacks on redpanda.py next may have the same question.